qa/suites/rgw/lua/tasks/0-install.yaml

```diff
@@ -3,11 +3,18 @@ tasks:
 - ceph:
 - openssl_keys:
 - rgw: [client.0]
-- tox: [client.0]    
+- tox: [client.0]
 
 overrides:
   ceph:
     conf:
       global:
         osd_min_pg_log_entries: 10
         osd_max_pg_log_entries: 10
+  install:
+    ceph:
+      extra_system_packages:
+        rpm:
+        - luarocks
```

Comment:
@yuvalif hi Yuval, i am very sorry for introducing the regression. 

luarocks depends on `liblua5.1-dev` or `liblua5.2-dev` or ``liblua5.3-dev` . see https://packages.ubuntu.com/noble/luarocks. if we want to use lua5.3, probably we should install `liblua5.3-dev` as well?

do you think it makes sense to "Suggest" `liblua5.3-dev` along with `luarocks` ?

Reply:
@tchaikov thanks for the input!
now passing teuthology: https://pulpito.ceph.com/yuvalif-2025-01-16_14:58:32-rgw:lua-wip-yuval-69473-distro-default-smithi/